charts/kube-otel-stack/scrape_configs.yaml

```diff
@@ -112,6 +112,14 @@
     source_labels:
     - id
     - pod
+  - action: replace
+    regex: "kube_service_info;(.*)"
+    replacement: "${1}"
+    separator: ";"
+    source_labels:
+    - __name__
+    - service
+    target_label: k8s_service_name
```

Comment:
@jaronoff97 @moh-osman3 this doesn't seem to currently do the trick... is a better place to do this in the processor itself vs the relabel config?

Reply:
I would put this in the attributes processor, the prometheusreceiver does some funky things iirc https://github.com/lightstep/otel-collector-charts/blob/main/charts/kube-otel-stack/values.yaml#L216-L226